SystemInfo/Collector/ConfigurationSymfonyKernelSystemInfoCollector.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/**
+ * File containing the ConfigurationSymfonyKernelSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * Collects information about the Symfony installation we are using.
+ */
+class ConfigurationSymfonyKernelSystemInfoCollector implements SystemInfoCollector
+{
+    /**
+     * Symfony environment.
+     *
+     * "dev" or "prod".
+     *
+     * @var string
+     */
+    private $environment;
+
+    /**
+     * True if Symfony is in debug mode.
+     *
+     * @var bool
+     */
+    private $debugMode;
+
+    /**
+     * An array containing the active bundles (keys) and the corresponding namespace.
```

Comment:
You should specify that the key is the bundle's name. An example of an array row would also help, I think.